src/main/java/com/redhat/devtools/lsp4ij/LSPFileSupport.java

```diff
@@ -36,7 +37,7 @@
  * LSP file support stored in the opened {@link PsiFile} with key "lsp.file.support"
  * which manages and caches LSP codeLens, inlayHint, color futures.
  */
-public class LSPFileSupport implements Disposable {

```

Comment:
@ApiStatus.Internal ?

Reply:
I set it as internal, for the moment I don't see some usecase to use it, we will see...